### ISSUE TITLE
DASH Widevine - First frame freeze

### ISSUE THREAD

**Issue by nicklasl** [google/ExoPlayer#4139]
### Issue description
When playing a linear/live stream the player will render the first frame (and go to STATE_READY) but it takes 1-2 seconds before the video starts. This happens for encrypted live/linear video streams only. We don't see the issue in (encrypted) VOD content and not for unencrypted live/linear streams.

### Reproduction steps
Demo app: Added the test content in the assets structure (with name, uri, drm_scheme & drm_license_url) but had to modify how to build the media source.
```
DefaultDashChunkSource.Factory chunkSourceFactory = new DefaultDashChunkSource.Factory(mediaDataSourceFactory);
DashMediaSource.Factory factory = new DashMediaSource.Factory(chunkSourceFactory, mediaDataSourceFactory);
MediaSource  mediaSource = factory.createMediaSource(uris[0], mainHandler, null);
```

### Link to test content
Separate email

### Version of ExoPlayer being used
2.7.1

### Device(s) and version(s) of Android being used
All that we've tested. Samsung Galaxy S7 (7.0), S8 (8.0) and Nexus 6P (8.1)

### A full bug report captured from the device
Separate email

**Comment by jboisjo** [comment]
We are having the same issue!

**Comment by jboisjo** [comment]
In older version of exo, we didn't have this issue. And also, when we remove the audio track in AudioRenderer, everything works well but there's no sound ofc.

**Comment by DWinnicki** [comment]
I have the same issue.

**Comment by ojw28** [comment]
If you can reproduce this issue then you can probably work out where the issue was introduced fairly easily (by trying different releases, and, ideally, by binary searching within the changes that went into the affected release). That would help diagnose the problem.

**Comment by jboisjo** [comment]
It's clearly a Player's DRM issue. Player shows first frame instead of showing the display when the player is running and not ready to play.

**Comment by ojw28** [comment]
As above, please help us to help you by trying to work out where the issue was reproduced. Restating the problem does not help with this. Thanks!

**Comment by jboisjo** [comment]
this has been resolved in 2.8.0 :D :D 

good job !
